pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -424,8 +425,6 @@ func (r *PerformanceProfileReconciler) Reconcile(ctx context.Context, req ctrl.R
 	return ctrl.Result{}, nil
 }
 
-func (r *PerformanceProfileReconciler) isMixedCPUsEnabled(profile *performancev2.PerformanceProfile) bool {
-	if !r.FeatureGate.Enabled(apiconfigv1.FeatureGateMixedCPUsAllocation) {
```

Comment:
bad rebase?

Reply:
Nope. this check moved down below.
See: https://github.com/openshift/cluster-node-tuning-operator/pull/1057/commits/213c596a977eba8c8c15359f00c0ddaf52b3c9ea